src/js/components/DataChart/doc.js

```diff
@@ -33,25 +33,39 @@ const thicknessType = PropTypes.oneOfType([
 ]);
 
 const chartType = PropTypes.shape({
-  key: PropTypes.string,
-  keys: PropTypes.arrayOf(
-    PropTypes.shape({
-      key: PropTypes.string,
-      color: colorType,
-    }),
-  ),
-  a11yTitle: PropTypes.string,
-  bounds: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.number)),
-  color: colorType,
+  property: PropTypes.oneOfType([
```

Comment:
While trying to learn the component API from the examples, I'm having a hard time understanding what `property` means as it has mixed context of react property or grommet properties. I think naming it `metric` will be more clear.

Reply:
I had a similar initial reaction. 

I so think we need to stay broader that `metric` because metric or measure infers a unit of measurement. In addition to units of measurement, we need this property to be able to contain categorical information such as 'animals', 'case status', 'order type', etc. I had suggested alternatives such as `attribute`, `dataAttribute`, `dataSeries`, `series`... I dig back through my notes to see if there are others.

Reply:
`series` will be the most consistent with other chart frameworks.

Reply:
The key concepts I was exploring was:
- `data` is the raw array of javascript objects
- `property` is how we decorate each property of the data objects that we will be showing. So, this includes how we render the value, what color we are using for it, etc. We use this for the chart coloring, point style, hover detail content, 
- `chart` is how we want to draw a data property, we can have multiple ways of drawing a property, area, line, point, etc.

Reply:
I chose `property` because that's what we use for DataTable: https://v2.grommet.io/datatable#columns

Reply:
For me `keys` can still mean many other things, while `series` context in Chart is pretty straight forward and self-explanatory.

Reply:
Would you want to align the term across DataTable and List, since it is conveying similar meaning?

Reply:
changed to `series`

Reply:
Thanks! Can we also change doc.js to use `series` instead of `property`?